frontend/src/components/TrashPageLoggedOutView.tsx

```diff
@@ -0,0 +1,5 @@
+const TrashPageLoggedOutView = () => {
```

Comment:
same as the above comment about the generic no-access component for logged out state